kubevirtci

```diff
@@ -276,7 +276,7 @@ EOF
 
 	${_kubectl} create -f $job_yaml
 	echo "-----------Waiting for curl job to complete"
-	${_kubectl} wait job curl-test-job -n default --for condition=Complete --timeout=5m
+	${_kubectl} wait job curl-test-job -n ${lb_namespace} --for condition=Complete --timeout=5m
```

Comment:
You also need to specify the ${lb_namespace} on lines
281, 284, 286
because the job pod is created in ${lb_namespace}

Reply:
Thanks! 286 is not needed since the namespace is part of the yaml file,

Reply:
Ah, of course :+1: